.github/scripts/print_pr_details.py

```diff
@@ -0,0 +1,55 @@
+# .github/scripts/print_pr_details.py
+from github import Github
+import os
+
+# Initialize the GitHub client
+token = os.getenv('GITHUB_TOKEN')
+g = Github(token)
+
+# Get repository information from environment variables
+repo_name = os.getenv('GITHUB_REPOSITORY')
```

Comment:
The PR number extraction from GITHUB_REF is fragile and could fail if the ref format changes. Consider adding error handling or validation.